tools/mmp/aot.cs

```diff
@@ -235,9 +235,10 @@ public void Compile (IFileEnumerator files)
 
 			if (IsRelease) {
 				// mono --aot creates .dll.dylib.dSYM directories for each assembly AOTed
-				// There isn't an easy was to disable this behavior, so clean up under release
+				// There isn't an easy was to disable this behavior
+				// We move them (cheap) so they can be archived for release builds
 				Parallel.ForEach (filesToAOT, ParallelOptions, file => {
-					if (RunCommand (DeleteDebugSymbolCommand, new [] { "-r", file + ".dylib.dSYM/" }, monoEnv) != 0)
+					if (RunCommand (MoveDebugSymbolCommand, new [] { file + ".dylib.dSYM/", Path.Combine (Path.GetDirectoryName (file), "..", "..", ".."), },  monoEnv) != 0)
```

Comment:
Why not use `Directory.Move`?

Reply:
It should work. I've re-used the existing logic, which did not use `Directory.Delete`

Reply:
fixed
it's faster than spawning processes - even when done in parallel